web_src/css/repo.css

```diff
@@ -236,8 +236,8 @@
   }
 }
 
-.repository.file.list #repo-desc {
-  font-size: 1.2em;
+#repo-desc {
+  font-size: 16px;
```

Comment:
IIRC, you have added `gt-font-16` in #24812 , could the `gt-font-16` be used to replace this style?

Reply:
done